lib/element.ts

```diff
@@ -730,6 +719,16 @@ export class ElementArrayFinder extends WebdriverWebElement {
     return this.applyAction_(allowAnimationsTestFn);
   }
 }
+// TODO(juliemr): might it be easier to combine this with our docs and just
+// wrap each one explicity with its own documentation?
+WEB_ELEMENT_FUNCTIONS.forEach((fnName) => {
+  ElementArrayFinder.prototype[fnName] = function (...args: any[]) {
```

Comment:
do you need here `()` as you did for `ElementFinder` (`(ElementFinder.prototype[fnName])`)?